### DISCUSSION TITLE
我写了份文档，如何零成本从0到1，开发并上线一个对接了openai的机器人，希望一起学习

### DISCUSSION THREAD

**Discussion by wang-xiaowu** [ChatGPTNextWeb/NextChat#592] [Show and tell]
https://wang-xiaowu.github.io/posts/2a9d73ff/

**Comment by lkbgct** [comment]
与主题无关，请教高手，试了n中以cf worker做的代理，在ios捷径中与AI聊，但是老是被CF 防火墙托管质询 安全级别掐断，在CF设置了放行都不行，如何解决？我还是觉得用捷径比装那些APP方便，可以自己加官方参数，自由度高，且安全。

**Reply by wang-xiaowu** [reply]
抱歉ios我不太熟悉
关于您说的第二点，我目前用的windows系统，我看到utools已经有人开发出对应插件了

**Comment by lkbgct** [comment]
嗯，谢谢！
